pkg/controller/machine/controller.go

```diff
@@ -323,3 +323,11 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, name string) error {
 	}
 	return r.Client.Delete(ctx, &node)
 }
+
+func delayAfterDrainFail(err error) (reconcile.Result, error) {
```

Comment:
`delayAfterDrainFail` is contextual info orthogonal to what this function does

Reply:
We actually do the same operation in other places e.g https://github.com/openshift/cluster-api/pull/25/files#diff-b96ce5bf8686fb02d855188000ae1cb4L212 we could do the same here. If we want to have a function and unit test it can we rename this functions and the tests ones to something more neutral?